BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -151,8 +151,11 @@ def update #:nodoc:
       end
     end
 
+    unless (current_user.available_users.map{ |u| u.id } | @group.user_ids).include?(params[:group][:creator_id].to_i )
```

Comment:
Adding it to the whitelist produces:
`[2016-11-02 15:46:00 UTC] update() Updated by admin: creator_id : value(1 -> 9)`
Using `addlog_object_list_updated` is prettier so I used that:
```
[2016-11-02 15:44:19 UTC] Creator updated by admin: Removed: cabingley
[2016-11-02 15:44:19 UTC] Creator updated by admin: Added: admin```